contracts/dao/PancakeStaking.sol

```diff
@@ -188,4 +207,33 @@ contract PancakeStaking is OwnableUpgradeable, ReentrancyGuardUpgradeable {
     function setHarvestTimeGap(uint256 _harvestTimeGap) external onlyOwner {
         harvestTimeGap = _harvestTimeGap;
```

Comment:
add an event

Reply:
done in 49ec5670cb578e687c2bd3d9cd4fabf31f7d2863